apps/css-workshop/src/components/Checkbox.astro

```diff
@@ -0,0 +1,71 @@
+---
+type Props = {
+  isLoading?: boolean;
+  indeterminate?: boolean;
+  checked?: boolean;
+  disabled?: boolean;
+  variant?: 'eyeball';
+  basic?: boolean;
+  status?: 'negative' | 'positive' | 'warning';
+} & astroHTML.JSX.HTMLAttributes;
+
+const { status, isLoading, indeterminate, disabled, variant, basic, ...props } = Astro.props;
```

Comment:
`basic` doesn't need to be a prop at all.

It can be derived from [`Astro.slots`](https://docs.astro.build/en/reference/api-reference/#astroslots):

```js
const basic = Astro.slots.has('default');
```

Reply:
resolved this on a new branch. pr soon